src/main/kotlin/FetchBlockIdCommand.kt

```diff
@@ -0,0 +1,22 @@
+import kotlinx.cli.ExperimentalCli
+import kotlinx.cli.*
+import kotlinx.coroutines.runBlocking
+
+@OptIn(ExperimentalCli::class)
+class FetchBlockIdCommand(private val apiClient: MempoolClient = MempoolClient()) :
+    Subcommand("fetchBlockId", "Fetch block ID for a given block height") {
+
+    private val startHeight by option(ArgType.Int, shortName = "s", description = "Start Height").required()
+    var result: String = ""
```

Comment:
Why are you using a variable outside of the function (same in `FetchTxIdsCommand`)? You only need `val` fields directly inside the `execute()` function.